### PR TITLE
freebsd: fix for linux specific code prevent freebsd from running pro…

### PR DESCRIPTION
**Pull request by bsnyder788**
…perly

### CODE REVIEW

**Comment by bsnyder788** [comment]
Fixes #236

**Review by mdanzinger — APPROVED** [review]
LGTM

File: cmd/do-agent/config_filesystem.go [hunk 1]
```diff
@@ -8,12 +8,12 @@ import (
 const (
 	ignoredMountPointFlag = "--collector.filesystem.ignored-mount-points"
 	ignoredFSTypesFlag    = "--collector.filesystem.ignored-fs-types"
-	ignoredMountPoints    = `^/(rootfs/)?(boot|sys|proc|dev|host|etc|var/(lib|run)/docker/[^$]+|run/docker/[^$]+)($$|/)`
+	ignoredMountPoints    = `^/(rootfs/)?(boot|sys|proc|dev|host|etc|tmp|usr/(home|ports|src)|var/(audit|crash|log|mail|tmp)|var/(lib|run)/docker/[^$]+|run/docker/[^$]+)($$|/)`
```
Comment (bsnyder788) [inline cmd/do-agent/config_filesystem.go:11]: @mdanzinger I added a few more exclusions here for freebsd

**Review by mdanzinger — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -25,6 +25,8 @@ git_rev      = $(shell git rev-parse --short HEAD)
 git_tag      = $(subst v,,$(shell git describe --tags --abbrev=0))
 VERSION     ?= $(git_tag)
 
+dockerlogin = @echo "${DOCKER_PASSWORD}" | docker login -u "${DOCKER_USERNAME}" --password-stdin
```
Comment (bsnyder788) [inline Makefile:28]: @mdanzinger I had to make one more commit for the CI to pass. Just added a docker login to bypass the default rate limiting they put in place

**Review by mdanzinger — APPROVED** [review]

**Pull request merged**